### PR TITLE
Homepage article

### PR DESCRIPTION
**Pull request by rgbcolor**

### CODE REVIEW

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "Is your homepage as good as than your door sign?"
+comments: true
+author: Maria Matveeva
+twitter: "rgbcolor"
+social: true
+summary: "Ask good questions to solve for real user goals"
+published: true
+tags: design, observations, UX, research
```
Comment (rgbcolor) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: UX is actually User Experience

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "Is your homepage as good as than your door sign?"
```
Comment (brzpegasus) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: There's an extra "than" in the title; should be just "as".

Comment (acacheung) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: extra than

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+twitter: "rgbcolor"
+social: true
+summary: "Ask good questions to solve for real user goals"
+published: true
+tags: design, observations, user experience
+---
+
+Today is a federal holiday, and a day off for us at DockYard. On days like this, when I do not travel, I often go to the local library, just across the street. 
+
+Today gave me the perfect opportunity to see how easy it would be to find a specific bit of information(*is the library open today?*) on a library website.
```
Comment (brzpegasus) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: space between "information" and parenthesis?

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+![](/images/20150120/11-Framingham-Public-Library--Look-Here-First.jpg)
+
+It was almost obvious that “all libraries will be closed” today — even on this crowded page. I was distracted by the email sign-up notice, but the next thing I read was the closure notice. Not pretty, but close to functional.
+
+###12. Franklin 
+##0/3 
+
+![](/images/20150120/12-Town-of-Franklin--MA---Franklin-Public-Library.jpg)
+
+The public library website seems to be just a layer fo content on top of the Franklin local government site. Very confusing. No indication of closure.
```
Comment (rsocci) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: typo:  'fo' should be 'of'

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+![](/images/20150120/13-Holliston-Public-Library---Holliston--Massachusetts.jpg)
+
+I found the closure information quickly, but it did not stand out enough (for my taste) from other news.
+
+###14. Lexington 
+##3.5/3 
+
+![](/images/20150120/14-Cary-Memorial-Library.jpg)
+
+The fist indication of the library closure for MLK is in the featured image. It eventually changes to another feature, but there is a second indicator (Today’s hours: CLOSED) that makes the situation obvious. This fall-back “closed” notice got this site an extra .5 points on my scale.
```
Comment (rsocci) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: typo: 'fist' should be 'first'

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+##What did I miss?
+
+Of course, this it a very small test for a very specific user need. The sites that did well on my scale for “closing hours” may fail miserably at providing an answer to “what’s the street address?” or “can I renew by book online?”.
```
Comment (rsocci) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: is the question supposed to be 'can I renew my book online?'

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+This library was a particular disappointment, because I have used it many times and the collection and its building are delightful. I am assuming it is one of the better funded libraries, given its location in direct proximity to Harvard. While the amount of content this page needs to show may be far greater than what a smaller local library deals with, this is no excuse for making things difficult to find. I get the feeling that the structure of the site is based on internal objectives, rather than user goals. 
+
+Here’s where today’s closure information is, in fact, located:
+
+![](/images/20150120/07-b-c-Library---City-of-Cambridge--Massachusetts.jpg)
+
+I would have to do this:
+- “find library hours” on the homepage
+- select “holidays” among eleven listings
+- see on the list that that today, January 19th, is a holiday
```
Comment (brzpegasus) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: double "that"

File: source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+comments: true
+author: Maria Matveeva
+twitter: "rgbcolor"
+social: true
+summary: "Ask good questions to solve for real user goals"
+published: false
+tags: design, observations, user experience
+---
+
+Today is a federal holiday, and a day off for us at DockYard. On days like this, when I do not travel, I often go to the local library, just across the street.
```
Comment (rsocci) [inline source/posts/2015-01-20-is-your-homepage-as-good-as-your-door-sign.md]: On days like 'these'

**Comment by brzpegasus** [comment]
@rgbcolor Nice post! I like how it concludes.

**Comment by bcardarella** [comment]
@rgbcolor just as a general rule, it might be best to host those images on something like [imgur](http://imgur.com) as adding more images to the repo just increases the overall footprint for everyone

**Pull request merged**